deck/test/functional/cypress/integration/google/instance_types_regional.spec.js

```diff
@@ -7,15 +7,20 @@ describe('google: Regional Instance Type Distribution', () => {
 
   beforeEach(() => {
     registerDefaultFixtures();
-    cy.route('/credentials?expand=true', 'fixture:google/accelerator_zones/credentials.json');
-    cy.route('/images/find?*', 'fixture:google/shared/images.json');
+    cy.intercept('/credentials?expand=true', {
+      fixture: 'google/accelerator_zones/credentials.json',
+    });
+
+    cy.intercept('/images/find?*', {
+      fixture: 'google/shared/images.json',
+    });
   });
 
   const selectImage = () => {
     const imageDropdown = ReactSelect('v2-wizard-page[key=location] div.form-group:contains("Image")');
     imageDropdown.get();
     imageDropdown.toggleDropdown();
-    imageDropdown.type('ubuntu-1404-trusty-v20181002');
+    imageDropdown.type('ubuntu-1404-trusty-v20181002', { force: true });
```

Comment:
Same as up above - force needed?

Reply:
responded in https://github.com/spinnaker/spinnaker/pull/7062#discussion_r2078089881 🙂